app/javascript/components/recipes/new/list.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div>
+    <h2 class="h-7 font-sans font-normal text-2xl text-black flex-grow mx-2">
+      {{ placeholder }}
+    </h2>
+    <div v-if="!drag">
+      <div
+        v-for="element in elements"
+        :key="element"
+      >
+        <container
+          :text="element"
+          :svg="svg"
+          :input="input"
+          :color="color('elements', element)"
+        />
+      </div>
+    </div>
+
+    <div v-if="input">
+      <container
+        :text="$t('msg.recipes.cost')"
+        :svg="{}"
+        :input="false"
+        :color="false"
+        class="my-4"
+      />
+
+      <base-button
+        :elements="{
+          placeholder: $t('msg.recipes.more'),
+          color: 'bg-white text-black border-2 border-blue-600 align-center hover:bg-gray-300 h-12'
+        }"
+        @click="search"
+        class="w-full"
+      />
+    </div>
+
+    <div
+      v-if="drag"
+    >
+      <!--drag-and-drop
+        :elements="this.steps"
+        :svg="svg"
+        :forceupdate="this.forceupdate"
+      /-->
+      <div
+        v-for="step in this.steps"
+        :key="step"
+      >
+        <container
+          :text="step"
+          :svg="svg"
+          :input="input"
+          :color="color('steps', step)"
+          @del="deleteStep"
+          @edit="editStep"
+        />
+      </div>
+
+      <text-box
+        :placeholder="$t('msg.recipes.step')"
+      />
+
+      <div class="grid grid-cols-4">
+        <div />
+        <div />
+        <div />
+        <base-button
+          :elements="{
+            placeholder: $t('msg.recipes.addstep'),
+            color: 'bg-white text-black border-2 border-blue-600 align-center hover:bg-gray-300 h-12 -mx-2'
+          }"
+          @click="addStep"
+          class="w-full"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Container from './container.vue';
+import DragAndDrop from '../../base/base-drag-and-drop.vue';
+
+export default {
+  components: {
+    Container,
+    DragAndDrop,
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The "DragAndDrop" component has been registered but not used  vue/no-unused-components

Reply:
Si no se está usando ese componente entonces mejor sacarlo 😅

Reply:
Por ahora no se usa, pero se va a usar para el drag and drop que implemente después. Igual lo voy a sacar mientras no esté implementado